src/js/components/tablero/NumberCard.jsx

```diff
@@ -1,26 +1,34 @@
 import React from 'react';
+import { SortableElement, sortableHandle } from 'react-sortable-hoc';
 import PropTypes from 'prop-types';
 import './tablero.scss';
 
 function truncateString(value, length) {
```

Comment:
I suggested once using the lodash's version of truncate and it is used in TableCard and UnarchivePopout already